packages/react-examples/README.md

```diff
@@ -3,3 +3,71 @@
 **Examples and documentation for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
 
 This package is only meant to be consumed by variants of the Fluent UI documentation site and therefore has **NO GUARANTEE OF API STABILITY**.
+
+## Architecture
+
+This package contains various root folders within `src`
+
+```sh
+|- src/
+|--- e2e/
+|--- azure-themes/
+|--- react/
+|--- react-cards/
+|--- react-charting/
+|--- react-experiments/
+|--- react-focus/
+```
+
+1. `e2e/` contains helpers for e2e ? TODO
+2. rest of the folders contain particular package `stories` and documentation content for `public-docsite` and `public-docsite-resources`
+
+### Structure of documentation/storybook folders
+
+1. Folder name reflects a package name omitting the npm scope.
```

Comment:
If you use one of the previous suggestions, this is now redundant with the point above. And probably these two sections could just be combined.